src/app/(nextjs_migration)/(authenticated)/user/layout.tsx

```diff
@@ -36,6 +64,7 @@ const MainLayout = async (props: Props) => {
         src="/nextjs_migration/client/js/nav.js"
```

Comment:
Unrelated, but I do love a good sidebar.
Do we have planned work to tweak all these URLs in the future before release? Having paths at launch like /nextjs_migration/ feels&hellip; wrong?